apps/etterlatte-saksbehandling-ui/client/src/components/behandling/beregningsgrunnlag/OverstyrBeregningGrunnlag.tsx

```diff
@@ -20,14 +131,146 @@ const OverstyrBeregningGrunnlag = (props: {
           </Heading>
         </HeadingWrapper>
       </ContentHeader>
-      <ToDoWrapper>TODO - innhold kommer i EY-3006</ToDoWrapper>
+      {isSuccess(overstyrBeregningGrunnlag) && (
+        <>
+          {visFeil && feil.length > 0 && behandles ? <FeilIPerioder feil={feil} /> : null}
+          <FormWrapper>
+            {fields.length ? (
+              <Table>
+                <Table.Header>
+                  <Table.Row>
+                    <Table.HeaderCell />
+                    <Table.HeaderCell scope="col">Periode</Table.HeaderCell>
+                    <Table.HeaderCell scope="col">Utbetalt beløp</Table.HeaderCell>
+                    <Table.HeaderCell scope="col">Trygdetid</Table.HeaderCell>
+                  </Table.Row>
+                </Table.Header>
+                <Table.Body id="forminstitusjonsopphold">
+                  {fields.map((item, index) => (
+                    <OverstyrBeregningTableWrapper
+                      key={item.id}
+                      item={item}
+                      index={index}
+                      control={control}
+                      register={register}
+                      remove={remove}
+                      watch={watch}
+                      visFeil={visFeil}
+                      feil={feil}
+                      behandles={behandles}
+                    />
+                  ))}
+                </Table.Body>
+              </Table>
+            ) : null}
+            {behandles && (
+              <Button
+                type="button"
+                icon={<PlusCircleIcon title="legg til" />}
+                iconPosition="left"
+                variant="tertiary"
+                onClick={() => {
+                  setVisFeil(false)
+                  append([
+                    {
+                      fom: nesteFomDato(sisteFom, sisteTom),
+                      tom: undefined,
+                      data: {
+                        utbetaltBeloep: '0',
+                        trygdetid: '0',
+                      },
+                    },
+                  ])
+                }}
+              >
+                Legg til beregningsperiode
+              </Button>
+            )}
+            {behandles && (
+              <Button type="submit" size="small" onClick={handleSubmit(ferdigstillForm)}>
+                Lagre
+              </Button>
+            )}
+            {visOkLagret && <CheckmarkCircleIcon color={AGreen500} />}
```

Comment:
```suggestion
              <Button type="submit" size="small" onClick={handleSubmit(ferdigstillForm)} loading={isPending(persistOverstyrBeregningGrunnlag)}>
                Lagre
              </Button>
            )}
            {isSuccess(persistOverstyrBeregningGrunnlag) && <CheckmarkCircleIcon color={AGreen500} />}
```

og så bare fjerne timer-logikken og staten for `visOkLagret`, siden vi ikke har de samme forutsetningene som institusonsopphold på denne. Evt. bare holde det likt for likhets skyld, du kan vurdere :p

Reply:
men loading bør nok med uansett

Reply:
Tok med loading men lot timer være - minst mulig endring for å få ting ferdig :)